vanilla-framework/vanilla-framework-react
# 85: Add Table component with optional sortable prop

# Done
- Added [Table](https://docs.vanillaframework.io/en/base/table) component
- Includes optional prop for [Table Sortable](https://docs.vanillaframework.io/en/patterns/table-sortable)
- Includes TableRow and TableCell subcomponents
- Added stories to Storybook
- Added snapshot tests

# QA
- Pull code
- Run `yarn lint` and `yarn test` to ensure there are no linting or testing errors
- Run `./run serve --watch` and navigate to http://localhost:8102/
- Verify the default Table component in Storybook matches the [Vanilla docs](https://docs.vanillaframework.io/en/base/table)
- Verify the sortable Table component in Storybook matches the [Vanilla docs](https://docs.vanillaframework.io/en/patterns/table-sortable)
- Verify that sorting works as expected

## Note
- I understand the testing is really bad. I'll be working on implementing proper unit testing on all the components some time in the future (see #90) 

# Fixes
Fixes #85
